android-components/components/ui/tabcounter/src/main/res/values/colors.xml

```diff
@@ -6,4 +6,5 @@
     <color name="mozac_ui_tabcounter_default_tint">#FF272727</color>
     <color name="mozac_ui_tabcounter_private_tint">#FFFFFF</color>
     <color name="mozac_ui_tabcounter_default_text">#20123A</color>
+    <color name="mozac_ui_photonPurple60">#ff952bb9</color>
```

Comment:
Could we import `ui-icons` here to preserve a single source of truth for our colors?